frontend/src/scenes/billing/billingLogic.ts

```diff
@@ -234,9 +236,14 @@ export const billingLogic = kea<billingLogicType>([
                         message: `You have exceeded the usage limit for ${productOverLimit.name}. Please 
                             ${productOverLimit.subscribed ? 'increase your billing limit' : 'upgrade your plan'}
                             or data loss may occur.`,
+                        dismissKey: 'usage-limit-exceeded',
                     }
                 }
 
+                lemonBannerLogic({ dismissKey: 'usage-limit-exceeded' }).mount()
```

Comment:
Turns out I simply needed to manually mount this logic.

Reply:
Would you look at that: 

https://github.com/PostHog/posthog/assets/21014901/cf9abe54-6d2d-40d0-81fc-fac1472e4d28